README.md

```diff
@@ -36,7 +36,11 @@ You will need to install troposhere and some other Python packages:
     
     In the directory  ```<APP_DIRECTORY>/infrastructure/src``` 
     
-    Type the following:```python -m babel.devenvironment --lambda-function-bucket <s3-bucket-containing-lambda-function-zipped> --environment <env> --aws-accountId <aws-account-id> --region <aws-region>```
+    Type the following:
+    ```python -m babel.environmentbuilder --lambda-function-bucket <s3-bucket-containing-zipped-lambda-function> --environment <env> --aws-accountId <aws account id> --region <aws region> --galileo-accountId <galileo accountId> --galileo-region <galileo region> --galileo-topic <galileo topic>```
+
```

Comment:
This is too much configuration to deal with on the command line, where all of these are basically constants per-environment (unless the Galileo maintainers change them).

I think the user should specify *env* on the command line (int, test, live, or a throwaway environment like "test2" or "tims"), and the code should then pick the galileo account number (one of two), galileo region (always eu-west-1 for the foreseeable future), and galileo topic. If those values are documented on Confluence or somewhere, there should be a link to whatever that page is.

So for the user, the command would be something like

     python -m babel.environmentbuilder int
     python -m babel.environmentbuilder test
     python -m babel.environmentbuilder throwawaytest4

Reply:
@timsweetman ` python -m babel.environmentbuilder throwawaytest4` so in this particular case what is the environment.

Reply:
The environment is "throwawaytest4", and I'd expect this to use *int*'s settings for the Galileo topic and account.

Reply:
Have changed to this...
The lamba function bucket is the only thing that appears not to be a constant..based on your assumptions..

```python3 -m babel.environmentbuilder --env throwawaytest4 --lambda-function-bucket galileo-babel-lambda```

Reply:
@timsweetman just realised that still need to change the readme.md..